### ISSUE TITLE
Daterange component is difficult to use

### ISSUE THREAD

**Issue by alejojumat** [HealthCatalyst/Fabric.Cashmere#985]
Based on usability testing at HAS, we learned that the daterange component is difficult to use.  
<img width="1129" alt="Screen Shot 2019-09-20 at 4 22 07 PM" src="https://user-images.githubusercontent.com/55603085/65356992-c616cf00-dbc3-11e9-9b84-d1d00c9924a3.png">

While there was a high completion rate, people rated it as difficult to use and watching them struggle changing a date range confirms what we saw.

Based on what we saw, I think what can solve the issue is just changing some labels!

Where it says for example "Sep 2019", just change the label to "2019 (the current year)" since when you click on that you see years and the user can change the year accordingly.

Where it say "Today", replace that with the month that the user is on since the left right arrows are used to navigate the months in that year.

**Comment by andrew-frueh** [comment]
Thanks for the research on this @alejojumat and providing the UX data.  Totally on-board with your recommendation for a fix.  Let's give that a try and test again to see if usability improves!

**Comment by corykon** [comment]
I agree @alejojumat!
Related: https://github.com/HealthCatalyst/Fabric.Cashmere/issues/703

**Comment by andrew-frueh** [comment]
@alejojumat @corykon - the pull request that I just submitted adjusts the date picker header controls like this:
<img width="364" alt="Screen Shot 2019-09-21 at 10 29 21 AM" src="https://user-images.githubusercontent.com/22795893/65376711-894ce580-dc60-11e9-9738-db4637dc2653.png">

The year button on the right triggers the same year picker functionality as before.  I think this much clearer UX and hopefully addresses the frustration your users ran into.  Note that a user can click on the month label to jump to the current date.  I think this is useful functionality, but probably not totally obvious, so I included a tooltip on hover:

<img width="382" alt="Tooltip" src="https://user-images.githubusercontent.com/22795893/65376750-f06a9a00-dc60-11e9-8222-f2ced69dff87.png">
